src/backend/config.ts

```diff
@@ -39,11 +39,11 @@ const rollerProd: Roller = {
 };
 
 const lenkerDev: Eksternlenker = {
-    efSakUrl: 'https://ensligmorellerfar.dev.intern.nav.no/ekstern',
-    baSakUrl: 'https://barnetrygd.dev.intern.nav.no',
-    ksSakUrl: 'https://kontantstotte.dev.intern.nav.no',
+    efSakUrl: 'https://ensligmorellerfar.intern.dev.nav.no/ekstern',
+    baSakUrl: 'https://barnetrygd.intern.dev.nav.no',
+    ksSakUrl: 'https://kontantstotte.intern.dev.nav.no',
     aInntekt: 'https://arbeid-og-inntekt.dev.adeo.no',
-    gosys: 'https://gosys-q1.dev.intern.nav.no/gosys',
+    gosys: 'https://gosys-q1.intern.dev.nav.no/gosys',
```

Comment:
Gosys kjører i fss og trenger ikke å oppdateres
```suggestion
    gosys: 'https://gosys-q1.dev.intern.nav.no/gosys',
```